common/src/test/java/org/corfudb/common/util/CompatibilityVectorUtilsTest.java

```diff
@@ -0,0 +1,49 @@
+package org.corfudb.common.util;
+
+import com.google.protobuf.ByteString;
+import org.corfudb.common.util.CompatibilityVectorUtils.Feature;
+import org.junit.Test;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+/**
+ * A test class to test the features of {@link CompatibilityVectorUtils} class and its
+ * {@link CompatibilityVectorUtils.Feature} enum.
+ * <p>
+ * Created by cgudisagar on 5/14/21.
+ */
+public class CompatibilityVectorUtilsTest {
+
+    private final ByteString vectors = CompatibilityVectorUtils.getCompatibilityVectors();
+
+    /**
+     * Tests {@link Feature#isFeatureEnabled(int, ByteString)} with invalid parameters.
+     */
+    @Test
+    public void testFeatureInvalidValues() {
+        // Assert that negative feature's bit returns False as features start from 0 onwards.
+        assertThat(CompatibilityVectorUtils.isFeatureEnabled(-1, vectors)
+        ).isFalse();
```

Comment:
nit: Can you address the formatting here?

Reply:
Done, Thank you.